pkg/api/config.go

```diff
@@ -9,117 +9,117 @@ import (
 )
 
 type Config struct {
-	ImageOffer     string
-	ImagePublisher string
-	ImageSKU       string
-	ImageVersion   string
+	ImageOffer     string `json:"imageOffer,omitempty"`
+	ImagePublisher string `json:"imagePublisher,omitempty"`
+	ImageSKU       string `json:"imageSku,omitempty"`
+	ImageVersion   string `json:"imageVersion,omitempty"`
 
 	// for development
-	ImageResourceGroup string
-	ImageResourceName  string
+	ImageResourceGroup string `json:"imageResourceGroup,omitempty"`
+	ImageResourceName  string `json:"imageResourceName,omitempty"`
 
 	Certificates CertificateConfig
 
 	// container images for pods
-	MasterEtcdImage             string
```

Comment:
do you need a json tag on `Certificates CertificateConfig` above?

Reply:
done :)